### PR TITLE
[team_pyajagod_0]:Helloworld for testing pr

### PR DESCRIPTION
**Pull request by pyajagod**
### What?
brief: 这是一个简单的测试pr
author: pyajagod

### What changes are included in this PR?
当前pr修改了test/README.md,也修改了team为test，用户为pyajagod

### Are these changes tested?
对代码src/main.cpp进行了测试，成功输出hello world

### CODE REVIEW

**Comment by Light-City** [comment]
> Tesing pf Auther: [qwy_qwy666@163.com](mailto:qwy_qwy666@163.com)
> 
> ### What?
> 包含了一个团队目录、个人目录、个人学习内容。
> 
> ### What changes are included in this PR?
> 测试代码见src/main.cpp，里面编写了一个helloworld程序
> 
> ### Are these changes tested?

content confusion

**Comment by Light-City** [comment]
Please post a screenshot of the main program running on your computer and let me know that your machine can run C++ programs

**Comment by pyajagod** [comment]
> 

image
My computer can run cpp programs

**Comment by pyajagod** [comment]
> Please post a screenshot of the main program running on your computer and let me know that your machine can run C++ programs

image
My computer can run cpp programs

**Comment by pyajagod** [comment]
> remove .idea dir or add it into .gitignore.

I have removed it, thanks again for checking it out!

File: .DS_Store
[the remark this answers is not in the document]
Comment (pyajagod) [reply .DS_Store:1]: i have done it, please checkout it.

File: stl/test/pyajagod/src/main
[the remark this answers is not in the document]
Comment (pyajagod) [reply stl/test/pyajagod/src/main:1]: i have finished it

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

**Comment by pyajagod** [comment]
> The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

i have done it, please check it again.

**Comment by tigerwe** [comment]
LGTM

**Pull request closed without merging**